github/repos_commits.go

```diff
@@ -115,6 +115,13 @@ type CommitsListOptions struct {
 	ListOptions
 }
 
+// BranchCommit is the result of listing branches with commit sha.
```

Comment:
Let's please change this to `SHA` instead of `sha`.
(Also, please don't use force commits just to make things easier for reviewers.)